tests/e2e/env-e2e-test.sh

```diff
@@ -17,17 +17,18 @@ export GINKGO_OPTS="-v"
 # export KUBECONFIG=""
 
 # Must supply path to values file if different from testfiles/values.yaml
-export VALUES_FILE="testfiles/appmob-values.yaml"
+export VALUES_FILE="testfiles/test-AM-one.yaml"
 
 # USER MODIFICATION REQUIRED: must supply path to your cert-csi binary
-export CERT_CSI="/root/cert-csi/cert-csi"
+export CERT_CSI="/root/cert-csi"
 
 # [Optional] uncomment any modules you want to test
 # export AUTHORIZATION=true
 # export REPLICATION=true
 # export OBSERVABILITY=true
 # export AUTHORIZATIONPROXYSERVER=true
 # export RESILIENCY=true
+# export APPMOBILITY=true
```

Comment:
I would do APPLICATIONMOBILITY here since all of the other vars are fully spelled out

Reply:
fixed